src/main/java/com/epam/edumanagementsystem/student/rest/service/StudentService.java

```diff
@@ -2,26 +2,20 @@
 
 import com.epam.edumanagementsystem.student.model.dto.StudentDto;
 import com.epam.edumanagementsystem.student.model.entity.Student;
+import com.epam.edumanagementsystem.util.service.UserService;
 
 import java.util.List;
-import java.util.Optional;
 
 public interface StudentService {
 
     Student create(StudentDto studentDto);
 
-    String delete(StudentDto studentDto);
-
-    StudentDto updateField(StudentDto studentDto);
-
-    StudentDto update(StudentDto studentDto);
+    Student create(StudentDto studentDto, UserService userService);
```

Comment:
We should return DTO but not entity, and in method we should pass entity but not DTO